cwms-data-api/src/main/java/cwms/cda/api/enums/VersionType.java

```diff
@@ -0,0 +1,48 @@
+package cwms.cda.api.enums;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jdk.nashorn.internal.runtime.Version;
+
+import javax.xml.bind.annotation.adapters.XmlAdapter;
+import java.time.ZonedDateTime;
+
+@Schema(
+        name = "Version Type",
+        description = "Version type specifies the type of timeseries response to be received. Can be max aggregate "
+                + "or single version. Max aggregate cannot be ran if version date field is specified."
+)
+public enum VersionType {
+    MAX_AGGREGATE("MAX_AGGREGATE"),
+    SINGLE_VERSION("SINGLE_VERSION"),
+    UNVERSIONED("UNVERSIONED"),
+    UNDEF("UNDEFINED");
```

Comment:
If you rename UNDEF to UNDEFINED you won't need the String ctor argument or versionType field.  or the value() or getValue() methods.

Reply:
I can do this in a followup pr.